src/public/modules/forms/base/controllers/submit-form.client.controller.js

```diff
@@ -31,6 +39,14 @@ function SubmitFormController(FormData, SpcpSession, $window, $document, GTag) {
   vm.myform.isTemplate = Boolean(FormData.isTemplate)
   vm.myform.isPreview = Boolean(FormData.isPreview)
   vm.myInfoError = Boolean(FormData.myInfoError)
+  if (
+    FormData.isIntranetUser &&
+    ['SP', 'CP', 'MyInfo'].includes(vm.myform.authType)
+  ) {
+    Toastr.permanentError(
```

Comment:
Should we also disable the singpass/myinfo sign in button?

Reply:
I don't think so - it's possible to log in through a workaround, and savvy users might figure that out. the purpose is just to warn the majority of users who likely won't figure it out